scripts/update-license.js

```diff
@@ -22,7 +22,7 @@ const year = new Date().getFullYear();
 
 files.forEach(file => {
   const filePath = path.join(__dirname, '../', file);
-  if (supported.indexOf(path.extname(filePath)) > -1 && fs.existsSync(filePath)) {
+  if (fs.existsSync(filePath) && supported.indexOf(path.extname(filePath)) > -1) {
```

Comment:
```js
supported.indexOf(path.extname(filePath)) > -1
```
Could be 
```js
supported.includes(path.extname(filePath))
```

`Array.includes` is supported from node 6.x.x - and is more readable by skipping that messy `> -1` check.

Reply:
I tend to forget nice things are available because they aren't on the web.